package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "skatejs-named-slots",
   "build:global": "skatejsNamedSlots",
-  "version": "0.1.9",
```

Comment:
is removing the version value intentional? seems like the kind of thing npm won't like.